js/happyfoxchat.js

```diff
@@ -1,18 +0,0 @@
-    <!-- HappyFox Live Chat Script -->
-    window.HFCHAT_CONFIG = {
-      EMBED_TOKEN: "995a2db0-46bf-11e5-986c-5b521052aa24",
-      ACCESS_TOKEN: "a5ba09020f0d40ae9eb91e299800cd47",
-      HOST_URL: "https://www.happyfoxchat.com",
-      ASSETS_URL: "https://d1l7z5ofrj6ab8.cloudfront.net/visitor"
-  };
-
```

Comment:
I'm okay with removing it... It's clearly not getting used much. I care more about it on the deck.